test/e2e/play_kube_test.go

```diff
@@ -815,43 +815,109 @@ var _ = Describe("Podman play kube", func() {
 		inspect := podmanTest.Podman([]string{"inspect", getCtrNameInPod(pod), "--format", "'{{ .Config.Cmd }}'"})
 		inspect.WaitWithDefaultTimeout()
 		Expect(inspect.ExitCode()).To(Equal(0))
+		cmd := inspect.OutputToString()
+
+		inspect = podmanTest.Podman([]string{"inspect", getCtrNameInPod(pod), "--format", "'{{ .Config.Entrypoint }}'"})
+		inspect.WaitWithDefaultTimeout()
+		Expect(inspect.ExitCode()).To(Equal(0))
+		ep := inspect.OutputToString()
+
 		// Use the defined command to override the image's command
-		correctCmd := "[" + strings.Join(defaultCtrCmd, " ") + " " + strings.Join(defaultCtrArg, " ")
-		Expect(inspect.OutputToString()).To(ContainSubstring(correctCmd))
+		Expect(ep).To(ContainSubstring(strings.Join(defaultCtrCmd, " ")))
+		Expect(cmd).To(ContainSubstring(strings.Join(defaultCtrArg, " ")))
 	})
 
-	It("podman play kube test correct command with only set command in yaml file", func() {
-		pod := getPod(withCtr(getCtr(withCmd([]string{"echo", "hello"}), withArg(nil))))
+	// If you do not supply command or args for a Container, the defaults defined in the Docker image are used.
```

Comment:
s/Docker/Container/g

Reply:
I chose this because I copied these comments straight from the k8s docs. I'll update them though